src/main/java/net/sf/jabref/exporter/SaveDatabaseAction.java

```diff
@@ -288,7 +288,10 @@ private boolean saveDatabase(File file, boolean selectedOnly, String encoding) t
             String tryDiff = Localization.lang("Try different encoding");
             int answer = JOptionPane.showOptionDialog(frame, builder.getPanel(), Localization.lang("Save database"),
                     JOptionPane.YES_NO_CANCEL_OPTION, JOptionPane.WARNING_MESSAGE, null,
-                    new String[] {Localization.lang("Save"), tryDiff, Localization.lang("Cancel")}, tryDiff);
+                    // @formatter:off
+                    new String[] {Localization.lang("Save"), tryDiff,
+                            Localization.lang("Cancel")}, tryDiff);
+                    // @formatter:on
```

Comment:
Please try not no use these formatter tags as we want to have them as little as possible. A simple solution would be in this case to create the array in a line above using 

`String[] options = {Localization.lang("Save"), tryDiff, Localization.lang("Cancel")};`

instead.